control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -219,7 +219,7 @@ func buildOASContainerMain(image string, etcdHostname string, port int32) func(c
 			},
 			{
 				Name:  "NO_PROXY",
-				Value: fmt.Sprintf("%s,%s,registry.access.redhat.com,quay.io,registry.redhat.io,amazonaws.com", manifests.KubeAPIServerService("").Name, etcdHostname),
+				Value: fmt.Sprintf("%s,%s", manifests.KubeAPIServerService("").Name, etcdHostname),
```

Comment:
I added the `amazonaws.com` in b65dbfa2a639205a1f3a1f4a186d391117b25865 because we couldn't pull the openshift/tools image from quay without which made the conformance tests fail.

I tested this again and routing this through the proxy works just fine, as long as the guest cluster has nodes. I am not sure why it didn't work back then, maybe we didn't properly wait for the guest clusters nodes to be up?